plugin/trino-hive/src/test/java/io/trino/plugin/hive/TestHiveDynamicPartitionPruning.java

```diff
@@ -16,46 +16,31 @@
 import com.google.common.collect.ImmutableMap;
```

Comment:
there is `io.trino.plugin.hive.TestHiveDistributedJoinQueries#testJoinWithEmptyBuildSide` that should be moved